public/Lab-files/Lesson02/Challenges/blog-post.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Blog</title>
+</head>
+<body>
+    <h1>Meditation</h1>
+    
+    </body>
```

Comment:
Congrats on finishing this first challenge! 
One problem that I noticed was that you put almost all of your content outside the ```<body></body>``` tags.  HTML5 is pretty forgiving, so it probably rendered when you viewed the page. I would suggest that next time, you indent properly (you have an extra indentation on line 10, which is probably what caused you to miss it). Doing so will minimize the amount of errors. Keep working hard!

```javascript
<body>
     ///content
</body>
```